### PR TITLE
Tweaks the lavaland ruin behind Legion to be more of a lair

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
obraz

### Intent of your Pull Request
Gives the ruin a (subjectively) better look, replaces floor with the necropolis floor, adds nice decals.
The drake now behaves more like Vasgrul (moves slower, but not as slow), and has 2 drakeling guardians. All the mobs inside ruin don't attack each other now. Added 2 tarantulas. There's now also a free drakeling egg behind it all. For the loot, there's piles of different coins, 25 mythril, 30 diamond and 35 gold stacks and a free survival medipen.

### Why is this change good for the game?
Makes it more interesting to traverse the ruin

# Wiki Documentation
### Briefly describe your PR and the impacts of it, in layman's terms. 
Changes the dragon lair a bit, the drake now has 2 drakeling protectors.

### What general grouping does this PR fall under? 
mining ruin change

# Changelog
:cl:  
tweak: Tweaks the lavaland ruin behind legion
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Picture plz

**Comment by Cartlord** [comment]
Yo I just wanna know why a dragon hoard would have a medi-pen, good PR tho

**Comment by ChesterTheCheesy** [comment]
> 
> 
> Yo I just wanna know why a dragon hoard would have a medi-pen, good PR tho

It's what's left behind of the last person trying to kill the dragon with his sword/sord (that i named "Dragonslayer"). It's obviously not worth much to a hoarding dragon, so it's on the pile of "rejects" along with sord, bananium chair (bananium isn't gold so it was thrown away) and a random "epic artifact" that is just a worthless copy (its the blue "X" on the image).

**Comment by SomeguyManperson** [comment]
don't think this guaranteed spawn drake should be weaker than normal drakes regardless of where it is

**Comment by ChesterTheCheesy** [comment]
> 
> 
> don't think this guaranteed spawn drake should be weaker than normal drakes regardless of where it is

It's not weaker in any sense, it's slower so it doesn't leave its lair. People now just bait the drake outside, where it's easier to fight. It also has 2 drakelings that will attack any interlopers.

**Review by patpol4 — CHANGES_REQUESTED** [review]
Fix the double templar 
vivaldi_qnLcQfLh5i

Other then that this should be fine I think.
The drake has slowdown on it:
vivaldi_JrwjIpC7rx
A better solution for the luring the drake out problem would probably be coding the door to close behind you and you having to get another key while in there by killing the drake? That's a code solution though so out of scope of this pr.

I'll leave the balance aspect of this to @SomeguyManperson because I never really played miner much and am not knowledgeable here.

Also keep in mind this adds:
a dragon egg
golden violin
sord
random fake artifact spawner
miner medipen.

**Review by patpol4 — APPROVED** [review]
Map fine, balance up to hd

**Comment by SomeguyManperson** [comment]
drakelings weren't made to function as enemies so they won't use any attacks and have a lower than average aggro range, and this would effectively let someone get three of them since you can revive the two dead ones if the bodies survive the fight with a lazarus injector

**Comment by ChesterTheCheesy** [comment]
i'll get to doing something with the drakelings soon
